.github/workflows/test.yml

```
@@ -470,6 +470,8 @@ jobs:
         brew install bash coreutils
         # QEMU 9.1.0 seems to break on GitHub runners, both on Monterey and Ventura
         # We revert back to 8.2.1, which seems to work fine
+        git config --global user.name "GitHub Actions Bot"
+        git config --global user.email "action@github.com"
```

Does GitHub explicitly allow using this email address?

If not:
```suggestion
        git config --global user.email "nobody@localhost"
```